ssdp.go

```diff
@@ -529,36 +530,42 @@ func (s *Ssdp) inMSearch(st string, req *http.Request, sendTo string) {
 	if st[0] == '"' && st[len(st)-1] == '"' {
 		st = st[1 : len(st)-2]
 	}
-	mx := 1
+	mx := time.Second
 	if mxStr := req.Header.Get("MX"); mxStr != "" {
 		mxInt64, err := strconv.ParseInt(mxStr, 10, 0)
 		if err != nil {
-			mx = int(mxInt64)
+			s.logger.Warnf("Could not parse mxStr: %s", err)
+			return
```

Comment:
I know the spec requires that MX be valid... however, I think would I prefer us to fall back to a default rather than "fail".

The log message is still welcome. Something like "MX Header" instead of "mxStr" might be easier for someone reading the log output.

Reply:
Fixed